app/react/Library/helpers/requestState.js

```diff
@@ -80,7 +80,11 @@ export default function requestState(request, globalResources, calculateTableCol
         }),
       ];
       if (calculateTableColumns) {
-        const tableViewColumns = getTableColumns(documents, templates);
+        const tableViewColumns = getTableColumns(
+          documents,
+          templates,
+          documentsRequest.data.types || []
```

Comment:
As a comment: these type of defaults are usually best configured as default for the function's arguments to prevent having to create this conditionals every time you call the function.